tests/helpers.py

```diff
@@ -1,24 +1,36 @@
 from datetime import datetime, timedelta
 
+def chunks(xs, n):
+    n = max(1, n)
+    return (xs[i:i+n] for i in range(0, len(xs), n))
```

Comment:
You may return a list directly instead of casting the retun value
```suggestion
    return  [xs[i:i+n] for i in range(0, len(xs), n)]
```

Reply:
Done in d2a9d00